gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ProcedureRequestMapper.java

```diff
@@ -60,7 +59,7 @@ public ProcedureRequest mapToProcedureRequest(
         procedureRequest
             .setStatus(ProcedureRequestStatus.ACTIVE)
             .setIntent(ProcedureRequestIntent.PLAN)
-            .setAuthoredOnElement(getAuthoredOn(planStatement.getAvailabilityTime(), ehrComposition))
```

Comment:
Looking [at the spreadsheet](https://docs.google.com/spreadsheets/d/1pG59D2Et3m7i7ne3a8Le7b_JhEOPwehZCG2FUMLRdDA/edit#gid=0) I'm confused as to why we changed `ProcedureRequestMapper`?